### PR TITLE
[conf] Make Gunicorn worker timeout and graceful timeout configurable

### PR DESCRIPTION
**Pull request by wing2fly**
## What changes were proposed in this pull request?

Add config option for gunicorn server

## How was this patch tested?

maunal

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

**Comment by athithyaaselvam** [comment]
LGTM!

**Review by athithyaaselvam — APPROVED** [review]

File: desktop/core/src/desktop/conf.py [hunk 1]
```diff
@@ -196,6 +196,19 @@ def is_custom_jwt_auth_enabled():
   type=int,
   default=1)
 
+GUNICORN_WORKER_TIMEOUT = Config(
+  key="gunicorn_work_timeout",
```
Comment (amitsrivastava) [inline desktop/core/src/desktop/conf.py:200]: please keep the full name `gunicorn_work_timeout` -> `gunicorn_worker_timeout`. Even though it gets longer, it keeps the meaning correct and also consistent with the internal names.

**Review by amitsrivastava — CHANGES_REQUESTED** [review]

File: desktop/core/src/desktop/conf.py (hunk 1 above)
Comment (Harshg999) [reply desktop/core/src/desktop/conf.py:200]: +1

**Review by Harshg999 — COMMENTED** [review]
Add these configs in `hue.ini` and `pseudo_distributed.tmpl`?

**Comment by wing2fly** [comment]
> Add these configs in `hue.ini` and `pseudo_distributed.tmpl`?

Added, thanks for the catch.

**Review by amitsrivastava — APPROVED** [review]

**Pull request merged**